frontend/src/__tests__/cypress/cypress/e2e/applications/explore.cy.ts

```diff
@@ -21,4 +21,8 @@ describe('Explore Page', { testIsolation: false }, () => {
   it('card title should be visible', () => {
     cy.get('span').should('contain', 'Jupyter');
   });
+
+  it('card title should not be visible', () => {
+    cy.get('span').should('not.contain', 'RHOAI');
```

Comment:
the logic is primarily in the backend and we aren't testing that. Adding this app to `mockComponents` will reflect the card in the `Explore` tab.

Reply:
Can you check for better selector than `span`. Additionally, using `not.contain` might not be effective if the text includes something like 'RHOAI abc'. Could you modify it to specifically check for the exact text 'RHOAI'?

Reply:
You should totally mock the rhoai app card as one of the resources returned in status, it will always not display rhoai if not deployed in the backend, mock that call and get the value

Reply:
added a component to `mockComponents`